resources/buildstock.rb

```diff
@@ -269,6 +269,25 @@ def get_value_from_runner_past_results(runner, key_lookup, measure_name, error_i
   return
 end
 
+def get_values_from_runner_past_results(runner, measure_name)
+  require 'openstudio'
+  values = {}
+  success_value = OpenStudio::StepResult.new('Success')
+  runner.workflow.workflowSteps.each do |step|
+    next if not step.result.is_initialized
+
+    step_result = step.result.get
+    next if not step_result.measureName.is_initialized
+    next if step_result.measureName.get != measure_name
+    next if step_result.value != success_value
+
+    step_result.stepValues.each do |step_value|
+      values["#{step_value.name}"] = get_value_from_workflow_step_value(step_value)
+    end
+  end
+  return values
+end
```

Comment:
What do you think about replacing the `get_value_from_runner_past_results` with this one, so that we don't potentially hit bottlenecks in the future in other parts of the code? Doesn't seem like it would be that much more work, and it may yield some additional performance improvements.

Reply:
I replaced it. Can you take a look at 585516c?